package.json

```diff
@@ -53,6 +53,8 @@
     "babel-preset-env": "^1.7.0",
     "babel-preset-react": "^6.24.1",
     "coveralls": "^3.0.3",
+    "enzyme": "^3.9.0",
```

Comment:
I thought we are required to use react-testing-library for testing @chikeozulumba , you might want to revisit this.

Reply:
@Raymond-Osy 
We are not required to use a particular library.